### PR TITLE
[rllib] Use compress observations where replay buffers and image obs are used  in tuned examples

### PR DESCRIPTION
**Pull request by avnishn**
Many of our tuned examples using replay and atari wouldn't be able to run on clusters or single node machines because they'll oom without enabling this compression. 

Signed-off-by: avnish <avnish@anyscale.com>

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by gjoliver — CHANGES_REQUESTED** [review]
nit nit nit
can we write the comments more formally, like:
```Compress observations to make sure the size of the ReplayBuffer fits most computers. ```
something like that? not really good at wording, but you get the idea.
thanks.

**Comment by avnishn** [comment]
could I change 
"we should set this because few machines would be able to contain the replay buffers in memory otherwise" to 

"we should **set compress_observations to True** because few machines would be able to contain the replay buffers in memory otherwise"

is that descriptive enough?

**Comment by avnishn** [comment]
I addressed the comments, but during today's experiments realized that only pong examples need obs compression so I turned off the compression in the non pong examples that I changed before.

File: rllib/tuned_examples/apex_dqn/atari-apex-dqn.yaml [hunk 1]
```diff
@@ -27,4 +27,4 @@ apex-breakoutnoframeskip-v4:
         rollout_fragment_length: 20
         train_batch_size: 512
         target_network_update_freq: 50000
-        min_sample_timesteps_per_iteration: 25000
\ No newline at end of file
+        min_sample_timesteps_per_iteration: 25000
```
Comment (avnishn) [inline rllib/tuned_examples/apex_dqn/atari-apex-dqn.yaml:30]: just needed a newline

**Review by gjoliver — APPROVED** [review]
this is good trick.

**Comment by gjoliver** [comment]
@richardliaw please merge

**Pull request merged**